contexts/security-v3-unstable.jsonld

```diff
@@ -67,6 +67,69 @@
         }
       }
     },
+    "EcdsaSecp256k1RecoveryMethod2020": {
+      "@id": "https://w3id.org/security#EcdsaSecp256k1RecoveryMethod2020"
+    },
+    "EcdsaSecp256k1RecoverySignature2020": {
+      "@id": "https://w3id.org/security#EcdsaSecp256k1RecoverySignature2020"
+      "@context": {
+        "@version": 1.1,
```

Comment:
```suggestion
```

JSON-LD 1.1 does feature detection, so you don't need to declare the version anymore.

Reply:
To be sure that this is okay... are we using any features that could be misinterpreted by JSON-LD 1.0 processors?

<https://w3c.github.io/json-ld-api/#features> says:

> Publishers are encouraged to use the `@version` map entry within a context set to 1.1 to ensure that JSON-LD 1.0 processors will not misinterpret JSON-LD 1.1 features.

Reply:
@manu - I can find nothing in the [JSON-LD 1.1 Rec](https://www.w3.org/TR/json-ld11/) about feature detection, and `"@version": 1.1` in many examples there. Please link to the new optionality?

Reply:
@TallTed, it's more that all version 1.1 features are defined such that they are either compatible or would trigger an error if a processing using 1.0 mode hits them. The `@version` tag just makes that happen immediately instead of when the feature is hit. The JSON-LD WG decided that explicitly declaring `@version` was a bad idea and harms adoption, and made an effort to ensure that you can just use the features you want/need and, if they are incompatible with JSON-LD 1.0, then such a processor will throw an error.

Reply:
@dlongley - As I recall from my (unfortunately brief) involvement with the JSON-LD WG working on the 1.1 spec, the `@version` declaration was necessary to make relative URIs behave properly for some use cases (including at least one that concerned me and OpenLink), and the lack of that declaration did not trigger any processing error in 1.0-based processors but did break the functionality which was making use of relative URIs.

I am now quite concerned that the hard-won solution to that issue has apparently been discarded, and more that people who clearly have no problem including the `@version` declaration are being advised not to do so -- when it remains at least legal, and again, clearly is not blocking their adoption/use of JSON-LD 1.1.

Reply:
I guess I need to write some tests for this case... does VC Data Model require JSON-LD 1.1? does DID Core?

Reply:
@TallTed -- that sounds like its getting off topic. But there's errata for JSON-LD 1.0 around prefix / relative URI definitions -- and I believe there are generally other mechanisms that (may have been invented after your time with the WG) can be used to force the behavior you want without having to resort to using `@version`. For example, using the new `@prefix` keyword, which, IIRC, will also trigger a JSON-LD 1.0 error. Again, I'm not sure which case you're talking about specifically, but my memory is that the group made it possible to trigger errors with JSON-LD 1.0 by using JSON-LD 1.1 specific features (for whatever specific problem you wanted solved) instead of having to rely on a catch-all version declaration.

Reply:
@dlongley -- Our issue involved relative URIs as property identifiers being expanded by some concatenation rule, rather than following [RFC 3986 rules](https://datatracker.ietf.org/doc/html/rfc3986), when expanding relative to `@base`. See [(long!) discussion](https://github.com/json-ld/json-ld.org/issues/488).  Assuming the Playground is fully up-to-date for JSON-LD 1.1, it appears to me that later revisions to that spec re-broke the things we had gotten to work earlier.  I'm checking with others to confirm my read on this, and then we'll log issues as appropriate.